traffic_ops/app/db/migrations/2021012800000000_remove_tm_path.sql

```diff
@@ -0,0 +1,23 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
```

Comment:
fancy. this could probably use a comment to explain what's going on

Reply:
I think that would be beneficial. I can see the updated information in the SQL table. It's removes the `tm_path` value from the `stats` object on _goose up_. If you need to revert with _goose down_, it subsequently adds a `tm_path` value back into the `stats` object with the value "/api/4.0/cdns/\<cdn name\>/snapshot"